packages/core/src/components/account/orders/MyAccountOrderDetails/MyAccountOrderDetails.tsx

```diff
@@ -0,0 +1,63 @@
+import {
+  Button as UIButton,
+  Badge as UIBadge,
+  Icon as UIIcon,
+  IconButton as UIIconButton,
+} from '@faststore/ui'
+import MyAccountStatusCard from 'src/components/account/orders/MyAccountOrderDetails/MyAccountStatusCard'
+import MyAccountOrderedByCard from './MyAccountOrderedByCard'
+import MyAccountDeliveryCard from './MyAccountDeliveryCard'
+import MyAccountPaymentCard from './MyAccountPaymentCard'
+import MyAccountSummaryCard from './MyAccountSummaryCard'
+
+import styles from './section.module.scss'
+
+export interface MyAccountOrderDetailsProps {
+  orderId: string
+}
+
+export default function MyAccountOrderDetails({
+  orderId,
+}: MyAccountOrderDetailsProps) {
+  return (
+    <div className={styles.page} data-fs-order-details>
+      <header data-fs-order-details-header>
+        <div data-fs-order-details-header-title>
+          <UIIconButton
+            size="small"
+            aria-label="Go back"
+            icon={<UIIcon name="ArrowLeft" />}
+          />
+          <h2>Order #{orderId}</h2>
```

Comment:
Why `h2`? Semantically, it only makes sense to use `h2` if we have a `h1` before somewhere; if not, it should be `h1`. WDYT?

Also, we might create a data-fs attr here and use it in the style file.